geomesa-accumulo/geomesa-accumulo-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/GeoMesaAccumuloInputFormat.scala

```diff
@@ -176,32 +178,13 @@ class GeoMesaAccumuloInputFormat extends InputFormat[Text, SimpleFeature] with L
   override def getSplits(context: JobContext): java.util.List[InputSplit] = {
     init(context.getConfiguration)
     val accumuloSplits = delegate.getSplits(context)
-    // fallback on creating 2 mappers per node if desiredSplits is unset.
-    // Account for case where there are less splits than shards
-    val groupSize = if (desiredSplitCount > 0) {
```

Comment:
we should deprecate desiredSplitCount and log a warning if it's being set (or since this is 1.3.0 we could just delete it)